### ISSUE TITLE
Contacts read as negative value when using karma

### ISSUE THREAD

**Issue by chummer5a** [chummer5a/chummer5a#95]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_
_Originally opened as https://github.com/chummer5a/chummer5/issues/21_

----

```
What steps will reproduce the problem?
1. Use all contact points (CHA X 3)
2. Increase connection or loyalty by one point (-1 Karma)
3. Contacts read -1

What is the expected output? What do you see instead?
Perhaps increment the contact total and annotate how many Karma are used?

What version of the product are you using? On what operating system?
0.0.5.139 via Wine on Linux

Please provide any additional information below.
This would be more of an enhancement than anything.  It is however confusing 
verifying where those extra couple Karma went.

```

Original issue reported on code.google.com by `j...@privmail.me` on 28 Oct 2014 at 6:03

**Comment by chummer5a** [comment]
Add additional labels for karma expenditure on contacts.

**Comment by chummer5a** [comment]
Resolved in 5.162.
